ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt

```diff
@@ -93,8 +93,6 @@ class RedirectBindingVerifier(private val response: IdpRedirectResponseDecorator
      * 3.4.6 Error Reporting
      */
     private fun verifyHttpRedirectStatusCode() {
-        // TODO remove the 200 check when "Manually change DDF IdP to respond with 302/303 status
```

Comment:
❓ See above question.

Reply:
❗️ Either leave a comment about allowing the 200 response code, or take out the extra check (`response.httpStatusCode != HttpStatusCodes.STATUS_CODE_OK`)